### PR TITLE
Fpm Sync API

### PR DESCRIPTION
**Pull request by AbrarNitk**

### CODE REVIEW

File: apis/sync.ftd [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+- Request Body
+
+-- ft.code:
+lang: json
+{
+    "package_name": "<Name of the package>",
+    "files": [
+        {
+            "mode": "Add/Update",
```
Comment (amitu) [inline apis/sync.ftd:58]: Rename "mode" to "action".

File: apis/sync.ftd [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+            "path": "<Absolute file path>",
+            "content": "file content",
+            "version": "file version which fpm store sent earlier"
```
Comment (amitu) [inline apis/sync.ftd:61]: Remove this, we will get version from latest-ftd.

**Comment by amitu** [comment]
@AbrarNitk what's the status of this PR?

**Comment by AbrarNitk** [comment]
@amitu It needs more work, will do it tomorrow.

**Pull request closed without merging**